bindata/network/multus/005-cron-schedule-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{.CronSchedulerConfigmap}}
+  namespace: openshift-multus
+  annotations:
+    kubernetes.io/description: |
+      Configmap containing user customizable cronjob schedule
```

Comment:
Nit, update to: 

> Configmap containing user customizable schedule for the Whereabouts ip reconciler using a cron expression

Reply:
Can you update this to include that it's for a whereabouts reconciler cron? thanks!